test/resources/resourceConfig.spec.js

```diff
@@ -38,7 +38,7 @@ describe('Build endpoint', () => {
     const endpoint = 'http://new.shoutem.com';
     const params = { x: 1 };
 
-    const expectedEndpoint = 'http://new.shoutem.com';
+    const expectedEndpoint = 'http://new.shoutem.com/?x=1';
```

Comment:
same here. 
Also, can we add a test for the opposite behavior (when the condition to use unused props is set to true)?